client/components/domains/transfer-domain-step/transfer-domain-precheck.jsx

```diff
@@ -173,11 +175,9 @@ class TransferDomainPrecheck extends React.Component {
 			);
 		}
 
-		if ( loading && ! isStepFinished ) {
```

Comment:
What I meant is just remove the `! unlockCheckClicked` check. :)

Reply:
Ah...ok. Sorry...I misunderstood.